v2/samples/apimanagement/v1api20220801/v1api20220801_authorizationprovidersauthorization.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: apimanagement.azure.com/v1api20220801
+kind: AuthorizationProvidersAuthorization
+metadata:
+  name: sampleapimauthorization
+  namespace: default
+spec:
+  authorizationType: "OAuth2"
+  error:
+    code: "samplecode"
+    message: "samplemessage"
+  oauth2grantType: "AuthorizationCode"
+  status: "samplestatus"
```

Comment:
Is this field required for the sample to pass? I naïvely think that the `status` field will convey a status and is not something that the end user will ever set? See my question elsewhere about if this field should just be marked readonly and thus not visible on `spec` at all